### PR TITLE
Drop text pointing to previous repo/version

### PR DESCRIPTION
**Pull request by jcpunk**
#### Pull Request (PR) description
The README notes information about versions under the old namespace.  Cleaned up.

### CODE REVIEW

**Review by aboe76 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ with systemd.
 
 ### Requirements
 
-* Puppet 4.6.1 or later.  Puppet 3 was supported up until release 0.2.0.
+* Puppet 4.6.1 or later.
```
Comment (ekohl) [inline README.md:44]: Looks like it's really 5.5.8 or later.

https://github.com/voxpupuli/puppet-chrony/blob/679302f6ff7f13a196e26c23492a08259158922c/metadata.json#L49

Given it's in metadata.json, is this really needed at all?

**Review by ekohl — COMMENTED** [review]
I always appreciate PRs fixing these kind of things.

File: README.md (hunk 1 above)
Comment (jcpunk) [reply README.md]: Makes sense to me, I'll adjust.

**Review by ekohl — APPROVED** [review]
@aboe76 do you agree?

**Comment by aboe76** [comment]
Yes that's a good idea, no need to repeat yourself.

**Review by aboe76 — APPROVED** [review]

**Pull request merged**